content/updates/2020-05/3-visibility-on-jobs.md

```diff
@@ -0,0 +1,11 @@
+# More visibility on all the jobs
+::: meta-data type="Improvement" features="Productivity, Process tracker" available="mid-May" link-to-doc="../articles/monitor-jobs.html#how-to-monitor"
+:::
+
+Say welcome to a brand new permission called `View all jobs in the Process Tracker`! It will allow you to see all jobs in the Process Tracker, even the ones that you didn't launch. Really neat if you are an admin and need to have a quick overview of what's running in the PIM. :wink:
+
+The previous `View Process Tracker` permission still exists and can now be combined with this new permission.
```

Comment:
Maybe we could remove this part..? Isn't it too detailed?

Reply:
We can. 
```suggestion
```